sentry-ruby/lib/sentry/span.rb

```diff
@@ -19,7 +19,7 @@ class Span
 
 
     attr_reader :trace_id, :span_id, :parent_span_id, :sampled, :start_timestamp, :timestamp, :description, :op, :status, :tags, :data
-    attr_accessor :span_recorder
+    attr_accessor :span_recorder, :transaction
```

Comment:
Ah nice, today I learned about `attr_accessor` after several "wtf" and bad suggestions to include a missing "@" in front of `transaction` and `span_recorder` down below 😄

Reply:
yeah, `attr_*` helpers could be confusing at first 😂